### DISCUSSION TITLE
I cant find how to set a compression level in python API

### DISCUSSION THREAD

**Discussion by vgdh** [miurahr/py7zr#571] [Q&A] [answered]
I can't find how to set a compression level in python API
Can you please write an example in help area?

**Comment by miurahr** [comment, marked as answer]
You can see the Python standard manual  for reference of LZMA/LZMA2  compression filters. 
https://docs.python.org/3/library/lzma.html#specifying-custom-filter-chains

Here is an example tin  standard manual  to set  compression level. 

```
my_filters = [
    {"id": lzma.FILTER_DELTA, "dist": 5},
    {"id": lzma.FILTER_LZMA2, "preset": 7 | lzma.PRESET_EXTREME},
]
```

Here is an example in py7zr manual

https://py7zr.readthedocs.io/en/latest/api.html#possible-filters-value

**Comment by vgdh** [comment]
thank you! ❤
